transit/rest_api/forms/pod_variance.py

```diff
@@ -30,8 +33,8 @@ class PODVarianceDetailsSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = PODVarianceDetails
-        fields = ['id', 'pod_variance', 'order_line_details', 'quantity']
-        read_only_fields = ['id']
+        fields = ['id', 'pod_variance', 'order_line_details', 'quantity', 'product_name', 'old_quantity']
```

Comment:
I'm not sure what's the impact of using model property on querying, for now we can keep it but it might be better to change it to `order__product__name`. We should profile it on larger dataset.